lib/improver/tests/spotdata/spotdata/test_WriteOutput.py

```diff
@@ -86,17 +86,9 @@ def test_invalid_method(self):
 
         method = 'unknown_file_type'
         msg = 'Unknown method ".*" passed to WriteOutput.'
-        with self.assertRaisesRegexp(AttributeError, msg):
+        with self.assertRaisesRegex(AttributeError, msg):
             Plugin(method, self.data_directory).process(self.cube)
 
-    def test_invalid_output_path(self):
```

Comment:
The reason that I removed this test was because it was causing a number of low-level warnings that looked like they were coming from the underlying C code:
```
.HDF5-DIAG: Error detected in HDF5 (1.10.1) thread 140639791306496:
  #000: H5F.c line 491 in H5Fcreate(): unable to create file
    major: File accessibilty
    minor: Unable to open file
  #001: H5Fint.c line 1247 in H5F_open(): unable to open file: time = Fri Jun  1 08:45:57 2018
, name = '/test_data.nc', tent_flags = 13
    major: File accessibilty
    minor: Unable to open file
  #002: H5FD.c line 809 in H5FD_open(): open failed
    major: Virtual File Layer
    minor: Unable to initialize object
  #003: H5FDsec2.c line 346 in H5FD_sec2_open(): unable to open file: name = '/test_data.nc', errno = 13, error message = 'Permission denied', flags = 13, o_flags = 242
    major: File accessibilty
    minor: Unable to open file
```
I don't think there would be a way of ignoring these messages. Also, I thought that this test was a bit unusual because essentially it's testing what happens when you try to save a file to the root directory, so it's more of a test of the file system, rather than plugin. Therefore, I thought that the simplest option would be to remove this test.

Reply:
Ok, that sounds fine to me.